README.md

```diff
@@ -237,7 +238,7 @@ Team Description Papers (TDPs) and Videos presented at the
 - **[Go for broke](https://drive.google.com/file/d/1VG49wvMTlj3t8SuXCBJVwZj_rMWNZaw6/view)** [PDF](https://drive.google.com/file/d/1BYrp3SLLn4-OMp8VMAXm3nQc92jr-UB1/view)
 - **[KicktronZ](https://drive.google.com/file/d/1LUr2r5q_8MV3QX-C1hOabRyJhmop__-T/view)** [PDF](https://drive.google.com/file/d/1oQkmzuPYxTvDL5VTjGHHgl99Q8ADZEFc/view)
 - **[KiloHertz](https://drive.google.com/file/d/14nHeRJRPVMEY0nkOxJmM-Z1W6Fb2m8Id/view)** [PDF](https://drive.google.com/file/d/1oPQruELQaHpV2GL3MmVr-vvQ_9lqN5bU/view)
-- **[LNX Robots](https://drive.google.com/file/d/14zEjYj4pQ90MZ_sjx4mUgHqnU_0Ctvk6/view)**
+- **[LNX Robots](https://drive.google.com/file/d/14zEjYj4pQ90MZ_sjx4mUgHqnU_0Ctvk6/view)** [PDF](https://lnxrobots.github.io/docs/2023/LNX_Robots_poster.pdf)
```

Comment:
Thank you. I have uploaded the PDF to our RCJ dedicated google drive. Can you update the link to https://drive.google.com/file/d/1WFm88cNSn49q9shjHiPXr8eSXZ06EX7D/view ?

Reply:
Sure, it should be now updated (5088093104fd394bf38a8e38c13636fe28252a30). Thank you!